xls/dslx/stdlib/std.x

```diff
@@ -374,6 +374,95 @@ fn msb_test() {
     assert_eq(u1:1, msb(s2:0b11));
 }
 
+
+// Returns the R LSbs of x.
+// `x` must be at least R bits wide.
+pub fn lsbs<R: u32, N: u32>(x: uN[N]) -> uN[R] {
```

Comment:
q: should we alias `lsb` and `msb` to those? (does it get optimized to the same thing?)